libsrc/atari5200/cputc.s

```diff
@@ -11,13 +11,13 @@
         .export         _cputcxy, _cputc
         .export         plot, cputdirect, putchar
         .import         gotoxy, _mul20
+        .import         conio_color
+        .importzp       screen_width, screen_height
         .importzp       ptr4
-        .import         setcursor
-
-        .constructor    screen_setup, 26
-        .import         screen_setup_20x24
-screen_setup    = screen_setup_20x24
 
+        .import         screen_setup
+        .constructor    screen_setup_constructor, 26
```

Comment:
Please remember to adjust https://github.com/cc65/wiki/wiki/Priority-of-constructors%2C-destructors%2C-and-interruptors accordingly.

Maybe just `initconio` (like on several other targets) instead of `screen_setup_constructor`.

BTW: Do you know if the prio is actually necessary? If yes, it would perhaps be nice if it were explained briefly in a comment.